extensions/smallrye-jwt/deployment/src/main/java/io/quarkus/smallrye/jwt/deployment/SmallRyeJwtProcessor.java

```diff
@@ -226,4 +245,95 @@ public boolean getAsBoolean() {
             return config.enabled();
         }
     }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
```

Comment:
Propose to move into a dedicated class like `SmallryJwtDevServiceProcessor` - even though we may not be starting a container, but may be we will launch Keycloak in some conditions